README.md

```diff
@@ -342,7 +346,7 @@ official [documentation](https://github.com/intel/linux-sgx). Please make sure t
 SDK version as denoted above in the list of requirements.
 
 For SGX SSL, just follow the instructions on the [corresponding
-github page](https://github.com/intel/intel-sgx-Sal). In case you are
```

Comment:
How the heck did this typo came in: from git blame it seems it was me who somehow screwed up, no idea how. Anyway, thanks for fixing ...